Cargo.toml

```diff
@@ -40,7 +40,7 @@ quickcheck = "0.9"
 criterion = "0.3"
 
 [features]
-default = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "jpeg_rayon"]
+default = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld", "jpeg_rayon"]
```

Comment:
Please add an extra configuration with _only_ the new features to `.travis.yml`.

Reply:
Done